### DISCUSSION TITLE
Overriding Add-On Settings

### DISCUSSION THREAD

**Discussion by SorenRogiers** [SFX-WoW/Masque#222] [Issues] [answered]
### Game Flavor

Retail

### Game Version

9.1

### Add-On Version

9.1
### Descriptioin

If you go to Masque skins settings you can specify settings for the addon called OmniBar.  Whenever I disable the swipe animation transparency in OmniBar it gets completely ignored by Masque and will show a swipe animation regardless. When going into a skin lua file (e.g. Dream.lua) changing:

```
Cooldown = {
....
Color {0,0,0,0}
...
}
```

Solves the issue.

### Steps To Reproduce

1. Have omnibar installed and set the swipe transparency to 0%
2. Apply a masque skin
3. Swipe animation will still show

### Screenshots

With Masque skin
https://gyazo.com/050cb65a0ea1ea0d4d4027d31abf7a36

Without a Masque skin
https://gyazo.com/930173301d5299f599ee2516218ef538

**Comment by StormFX** [comment, marked as answer]
Masque, at its core, is a library.  It doesn't compensate for add-on specific settings.  That's up to the add-on being skinned.  OmniBar (or the relevant Masque plug-in) will need to check if Masque is enabled and make appropriate adjustments.
